### DISCUSSION TITLE
[Feature] Jump to date on timeline from photo preview (calendar icon)

### DISCUSSION THREAD

**Discussion by lukanowa** [immich-app/immich#8261] [Feature Request]
### The feature

Hello,

Sometimes when I viewing searched photos I want to jump to timeline to date of viewing photo and see more photos from this date.

I think the calendar icon in photo details is perfect place for this feature (jump to timeline).

What You think?:)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #4115
